### DISCUSSION TITLE
Redirect to Page,Controll

### DISCUSSION THREAD

**Discussion by ruwansj** [serenity-is/Serenity#6765] [Help]
Hi,

I do want to redirect to another control or a page from the Update popup when certain conditions.  Before that, I want to display a message according to the condition and redirect it to a control.  Please guide me. I'm new to the Serenity framework. I could not find a solution through Google. 

public SaveResponse Update(IUnitOfWork uow, SaveRequest<MyRow> request,    [FromServices] IBusinessNatureSaveHandler handler)
{
    Message("Cannot Save");
   Redirect("Application/List");
}

**Comment by ali-h2010** [comment]
I have a similar problem. I want to redirect to a specific page instead of the dashboard. 
Could someone help me with that?

**Reply by VictorTomaili** [reply]
Please do not spam in every discussion

**Comment by muhammedyaman** [comment]
Do you mean a notification like system or you want to update a record and depending on its response you will redirect to another page?

**Comment by stixoffire** [comment]
You can use RedirectToAction("Index", MyController)  other methods .. depending on what you really want to do
